### PR TITLE
Document timestamp provider for HBase backend #887 [skip ci]

### LINKED ISSUE #887: Document timestamp provider for HBase backend
HBase currently uses millisec for timestamps.  Setting JanusGraph timestamp provider to anything else will have consequence. Let's document it.

### PR DESCRIPTION
**Pull request by jerryjch**
Fixes #887

Signed-off-by: jinghe <jinghe@us.ibm.com>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there an issue associated with this PR? Is it referenced in the commit message?
- [x] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [x] Have you ensured that format looks appropriate for the output in which it is rendered?
- [x] If this PR is a documentation-only change, have you added a `[skip ci]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by mbrukman** [comment]
@jerryjch — as you can see in https://github.com/JanusGraph/janusgraph/pull/1089.patch, your commits are not using your full name, which is required for the CLA verification.

Also, please use your full name in the `Signed-off-by` line, which comes from the same place (git config). Please see this doc for details: https://github.com/JanusGraph/janusgraph/blob/master/CONTRIBUTING.md#configure-your-repo-to-match-the-cla

File: docs/hbase.adoc [hunk 1]
```diff
@@ -78,10 +78,14 @@ Refer to <<config-ref>> for a complete listing of all HBase specific configurati
 
 When configuring HBase it is recommended to consider the following HBase specific configuration options:
 
-* *tablename*: Name of the HBase table in which to store the JanusGraph graph. Allows multiple JanusGraph graphs to co-exist in the same HBase cluster.
+* *storage.hbase.table*: Name of the HBase table in which to store the JanusGraph graph. Allows multiple JanusGraph graphs to co-exist in the same HBase cluster.
 
 Please refer to the http://hbase.apache.org/book/config.files.html[HBase configuration documentation] for more HBase configuration options and their description. By prefixing the respective HBase configuration option with `storage.hbase.ext` in the JanusGraph configuration it will be passed on to HBase at initialization time. For example, to use the znode /hbase-secure for HBase, set the property: `storage.hbase.ext.zookeeper.znode.parent=/hbase-secure`. The prefix allows arbitrary HBase configuration options to be configured through JanusGraph.
 
+[IMPORTANT]
+HBase backend uses millisecond for timestamps. In JanusGraph 0.2.0 and before, if the `graph.timestamps` property is not explicitly set, the default is MICRO.
```
Comment (mbrukman) [inline docs/hbase.adoc]: s/and before/and earlier/ ?

Comment (mbrukman) [inline docs/hbase.adoc]: Format `MICRO` with code formatting, please.

**Comment by jerryjch** [comment]
@mbrukman Updated based on your comments.  Thanks!

**Pull request merged**